sdk/internal/recording/sanitizer.go

```diff
@@ -401,8 +401,8 @@ func AddURISubscriptionIDSanitizer(value string, options *RecordingOptions) erro
 	return handleProxyResponse(client.Do(req))
 }
 
-// ResetSanitizers deletes any existing sanitizers from the test proxy.
-func ResetSanitizers(options *RecordingOptions) error {
+// ResetProxy deletes any existing sanitizers, matchers, and transforms from the current test proxy session
```

Comment:
Just a suggestion, trying to capture that resetting means going back to default settings:
```suggestion
// ResetProxy restores the proxy's default sanitizers, matchers, and transforms
```